### ISSUE TITLE
Bacula 11.0.6 broken

### ISSUE THREAD

**Issue by xchotard-talentsoft** [mkevenaar/chocolatey-packages#187]
Unable to install bacula with `--version 11.0.6` because it seems that the upstream source url & checksum have been changed.

## Expected Behavior
`choco install bacula -y --version 11.0.6` should work.

## Current Behavior
```
PS C:\> choco install bacula -y --version 11.0.6
Chocolatey v1.3.1
Installing the following packages:
bacula
By installing, you accept licenses for the packages.
Progress: Downloading bacula 11.0.6... 100%

bacula v11.0.6
bacula package files install completed. Performing other installation steps.
Attempt to get headers for https://www.bacula.org/download/10517/ failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://www.bacula.org/download/10517/'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (500) Internal Server Error."
Downloading bacula 64 bit
  from 'https://www.bacula.org/download/10517/'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://www.bacula.org/download/10517/'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (500) Internal Server Error."
The install of bacula was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\bacula\tools\chocolateyInstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - bacula (exited 404) - Error while running 'C:\ProgramData\chocolatey\lib\bacula\tools\chocolateyInstall.ps1'.
 See log for details.
```

## Possible Solution
Update `url64` and `checksum64` values in `chocolateyInstall.ps1` for the 11.0.6.
It seems like `https://www.bacula.org/download/10517` have been changed to `https://www.bacula.org/download/11049`.
I did not tested url32/x86 download neither other versions.

## Your Environment

* Package Version used: 11.0.6
* Operating System and version: Windows Server Standard 2022
* Chocolatey version: 1.3.1
* Install/uninstall gist:

**Comment by xchotard-talentsoft** [comment]
Tag `bacula-11.0.6` file `chocolatey-packages/automatic/bacula/tools/chocolateyInstall.ps1` new values:
```
$url64          = 'https://www.bacula.org/download/11049/'
$checksum64     = '4703fadbb0c53c677d3d08250e3d8a3aa1432a641c88758f579df88a368ac7c2''
```
I would like to do a pull request, but the layout of the repository is a bit confusing. This would be a fix only for the `bacula-11.0.6` tag.

**Comment by mkevenaar** [comment]
This should be fixed now
